dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/MetadataConverter.java

```diff
@@ -58,7 +59,7 @@ public MetadataRest convert(List<MetadataValue> metadataValueList) {
                 set = new TreeSet<>(Comparator.comparingInt(MetadataValueRest::getPlace));
                 mapOfSortedSets.put(key, set);
             }
-            set.add(valueConverter.convert(metadataValue));
+            set.add(converter.toRest(metadataValue, Projection.DEFAULT));
```

Comment:
Good catch. 

The reason this is different is just because I didn't take the extra time to refactor `MetadataConverter` into a `DSpaceConverter` which takes a projection argument. At the time I judged it might not be trivial, so skipped refactoring it...but now with fresh eyes I'm not seeing why that should be so hard.

I will take another look and see if I can pull this in line with the others.

Reply:
Brought in line with the others; see latest commit